cvat/apps/documentation/user_guide.md

```diff
@@ -169,18 +172,39 @@ Go to the [Django administration panel](http://localhost:8080/admin). There you
     The option helps to load high resolution datasets faster.
     Use the value from ``1`` (completely compressed images) to ``95`` (almost not compressed images).
 
-    **Select files**. Push this button to select files you want to annotate.
+    **Frame Filter**. Use this option to filter video frames.
+    For example, enter ``step=25`` to leave every twenty fifth frame in the video. Use this option on video files only.
+
+    **Select files or URL list**. If you select `` Remote`` in `` Source``, you'll see a field where you can enter
+    an URL adrresses (one URL per line).
+    Press `` Local`` or `` Share`` in the `` source`` field to choose some files
+    for anotation from your local PC or a network folder respectively.
 
     Push ``Submit`` button and it will be added into the list of annotation tasks.
-    Then, the created task will be displayed on dashboard:
+    Then, the created task will be displayed on a dashboard:
 
-    ![](static/documentation/images/image006.jpg)
+    ![](static/documentation/images/image006_DETRAC.jpg)
 
 1.  The Dashboard contains elements and each of them relates to a separate task. They are sorted in creation order.
     Each element contains: task name, preview, execution status, buttons, and one or more links.
     Each button is responsible for a specific function:
-    - ``Dump Annotation`` — download an annotation file from the task (xml format)
-    - ``Upload Annotation`` — upload an annotation file to the task (xml format)
+    - ``Dump Annotation`` — download an annotation file from a task. Several formats are available:
+      - [CVAT XML 1.1 for video](https://github.com/opencv/cvat/blob/develop/cvat/apps/documentation/xml_format.md#interpolation)
```

Comment:
Let's use relative links everywhere. For example in this case: ``cvat/apps/documentation/xml_format.md#interpolation``